.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation Issue
+about: 'An issue pertaining to the documentation. '
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**describe the issue**
+Is it a typo, or something that is missing from the documentation, etc. 
+
+**Link to the page containing the issue, if applicable**
```

Comment:
```suggestion
**Link to the page and section containing the issue, if applicable**
```
Users can link the exact section of page to help narrow things down more quickly.